sql/sql_statistics.h

```diff
@@ -154,6 +155,7 @@ class Histogram
     case SINGLE_PREC_HB:
       return ((uint) (1 << 8) - 1);
     case DOUBLE_PREC_HB:
+    case JSON:
       return ((uint) (1 << 16) - 1);
```

Comment:
It's ok for now, but in the future this code should be removed as the concept of `prec_factor` doesn't apply to Histograms that are stored as JSON.